src/components/navigation/problem-panel.tsx

```diff
@@ -5,6 +5,7 @@ import { CanvasComponent } from "../document/canvas";
 import { SectionModelType } from "../../models/curriculum/section";
 import { DocumentContentModelType } from "../../models/document/document-content";
 import { SectionToolbar } from "../document/section-toolbar";
+import clsx from "clsx";
```

Comment:
Library import should be at the top.

Reply:
:+1: Fixed in https://github.com/concord-consortium/collaborative-learning/pull/2486/commits/d37621f9ab623bdcd0908aef2a9b80d58e82921b